e2e/rbd.go

```diff
@@ -118,6 +117,7 @@ var _ = Describe("RBD", func() {
 			if cephCSINamespace != defaultNs {
 				err := createNamespace(c, cephCSINamespace)
 				if err != nil {
+					e2elog.Logf("failed to create namespace %v", err)
```

Comment:
why it is required? i think its already logged with a trace in `Fail(err.Error())`

Reply:
Added as part of debugging , let me remove it.